lib/ansible/modules/cloud/vmware/vmware_guest_facts.py

```diff
@@ -63,6 +63,18 @@
 '''
 
 EXAMPLES = '''
+
+# Gather facts
+  - name: gather the VM facts
+    vmware_guest_facts:
+      hostname: 192.168.1.209
+      username: administrator@vsphere.local
+      password: vmware
+      datacenter: vmdatacenter
```

Comment:
Can you rename datacenter as `ha-datacenter` which is default datacenter for standalone ESXi server ? This will very helpful to people who uses standalone ESXi and searching for Ansible playbook example for the same.